src/agent/agent.h

```diff
@@ -60,9 +60,11 @@ struct Agent {
         int port;
         char *controller_address;
         long heartbeat_interval_msec;
+        long heartbeat_threshold_msec;
```

Comment:
Adding a prefix of `controller_` should help distinguishing between the agent heartbeat and the threshold used to check with the heartbeat received from the controller:
```suggestion
        long controller_heartbeat_threshold_msec;
```

Reply:
done